lib/raven/configuration.rb

```diff
@@ -172,16 +172,29 @@ class Configuration
     # Errors object - an Array that contains error messages. See #
     attr_reader :errors
 
+    # Most of these errors generate 4XX responses. In general, Sentry clients
+    # only automatically report 5xx responses.
     IGNORE_DEFAULT = [
       'AbstractController::ActionNotFound',
+      'ActionController::BadRequest',
       'ActionController::InvalidAuthenticityToken',
+      'ActionController::InvalidCrossOriginRequest',
+      'ActionController::MethodNotAllowed',
+      'ActionController::NotImplemented',
+      'ActionController::ParameterMissing',
       'ActionController::RoutingError',
       'ActionController::UnknownAction',
+      'ActionController::UnknownFormat',
+      'ActionController::UnknownHttpMethod',
+      'ActionDispatch::Http::Parameters::ParseError',
+      'ActionView::MissingTemplate',
```

Comment:
@st0012 I'm not sure what the current state of the `v3` branch is, but the inclusion of `ActionView::MissingTemplate` here feels like a bug. While Rails does render a `406 Not Acceptable` when [`ActionView::MissingExactTemplate`](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb#L16) is raised, it renders a `500 Internal Server Error` when `ActionView::MissingTemplate` is raised (via its [default error handling](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb#L8)).

Reply:
yeah I think it's too aggressive to exclude the `ActionView::MissingTemplate` by default. will remove it in the upcoming version. thanks for the opinion 👍

Reply:
I've added https://github.com/getsentry/sentry-ruby/pull/1092 to remove it

Reply:
Thanks @st0012!!